### ISSUE TITLE
One of your three mods (SASSOT, Armor Restitched, Progressive Archery) is giving me issues with name tags (See attached video) 

### ISSUE THREAD

**Issue by GurningBoose** [SolipIngen/minecraft.sassot#69]
Hi, I'm using your mods in a modpack for Fabric 1.21 and, for some reason, after I added your mods, I could not rename name tags. I have tested renaming name tags with all of the mods separately and in combination with one another and they work fine, but once I add yours, name tags can no longer be renamed, nor does the anvil show the enchantment cost specifically for changing the name of the name tag. See the video below for an example. I also attached a .zip file of all the mods I am using. Thanks for taking the time :)

Video: https://github.com/user-attachments/assets/2ca4d90c-aa58-4551-b1cf-2c30c723b2a1

mods.zip: https://drive.google.com/file/d/1DEdXlfsi45yN44cGcRr7QEVwL9Yx9u3I/view?usp=sharing

**Comment by SolipIngen** [comment]
Hello, thank you for reporting this issue.

I have an immediate suspicion that the problematic mod is Armor Restitched. Only that mod changes anvil behaviour, which makes it so that renaming items does not cost any anvil levels. This will be confirmed and fixed by the next mod update, which should be within the next couple of days.

Just to make things tidier, would you please open this Issue on Armor Restitched's Issues page instead? I will close this thread here once that is done.

**Comment by SolipIngen** [comment]
Hello.

This issue has been addressed and fixed in the latest version of Armor Restitched (v1.2.1-1.21). You can find it on Modrinth and CurseForge, as usual.

I will now close this issue. Please feel free to install the mod and see if the issue indeed has been resolved. If it persists, you can open a new issue on Amor Restitched's Issues page.
